war/src/main/js/components/dropdowns/templates.js

```diff
@@ -45,7 +45,7 @@ function menuItem(options) {
   const tag = itemOptions.type === "link" ? "a" : "button";
 
   const item = createElementFromHtml(`
-      <${tag} class="jenkins-dropdown__item" href="${itemOptions.url}">
+      <${tag} class="jenkins-dropdown__item ${itemOptions.clazz ? itemOptions.clazz : ""}" href="${itemOptions.url}" id="${itemOptions.id}">
```

Comment:
```suggestion
      <${tag} class="jenkins-dropdown__item ${itemOptions.clazz ?: ""}" href="${itemOptions.url}" id="${itemOptions.id}">
```

JEXL supports Elvis operator. Untested.

Reply:
Updated it - will see if it passes unil tests

Reply:
Unit tests weren't happy sadly, reverted it.

Reply:
Does javascript support the Elvis operator? This is not jelly

Reply:
It doesn't - I've replaced it with `||` and it looks to be working fine. Also updated it to remove `href` being added when its a button rather than a link. Thanks!

Reply:
> This is not jelly

Ah, good point, it looked so similar to JEXL/Jelly that I thought it was that. Sorry for the noise!